lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/dao/DBOFileHandleDaoImplTest.java

```diff
@@ -712,4 +724,50 @@ public void testCreateBatch() {
 			assertEquals(ObjectType.FILE, message.getObjectType());
 		}
 	}
+	
+	@Test
+	public void testUpdateStorageLocationBatch() {
+		StorageLocationSetting oldStorageLocation1 = TestUtils.createExternalStorageLocation(creatorUserGroupIdL, "Old Storage Location");
+		StorageLocationSetting oldStorageLocation2 = TestUtils.createExternalStorageLocation(creatorUserGroupIdL, "Old Storage Location");
+		
+		StorageLocationSetting newStorageLocation = TestUtils.createExternalStorageLocation(creatorUserGroupIdL, "New Storage Location");
+		
+		Long oldStorageLocation1Id = storageLocationDao.create(oldStorageLocation1);
+		Long oldStorageLocation2Id = storageLocationDao.create(oldStorageLocation2);
+		Long newStorageLocationId = storageLocationDao.create(newStorageLocation);
+		
+		storageLocationsToDelete.addAll(Arrays.asList(oldStorageLocation1Id, oldStorageLocation2Id, newStorageLocationId));
+		
+		FileHandle fileHandle1 = TestUtils.createExternalFileHandle(creatorUserGroupId, idGenerator.generateNewId(IdType.FILE_IDS).toString());
+		FileHandle fileHandle2 = TestUtils.createExternalFileHandle(creatorUserGroupId, idGenerator.generateNewId(IdType.FILE_IDS).toString());
+		

```

Comment:
Add a FileHandle whose storageLocationId is the newStorageLocationId so that we check it doesnt't change